React_Notes_Block/src/pages/mainNotes/listNotes/ListNotes.jsx

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-unused-expressions */
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Typography, Grid, Box } from '@material-ui/core';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import Stack from '@mui/material/Stack';
+import WAHA_NOTES, { SHORT_DESC } from '../../../config/constants/initNoteData';
+import useStyles from './styled';
+import chooseNote from '../../../utils/ChooseNote';
+import callToEditNote from '../../../utils/CallToEditNote';
+import saveEditedNote from '../../../utils/SaveEditedNote';
+import deleteNote from '../../../utils/DeleteNote';
+import ButtonChange from '../../../components/notes/ActionButtons';
+import InputChange from '../../../components/notes/InputChange';
+import setToLocalStorage, {
+  getToLocalStorage,
+} from '../../../utils/localStorage/SetGetLocStor';
+
+const ListNotes = ({ sentAbout }) => {
+  const [noteList, setNoteList] = useState(WAHA_NOTES);
+  const classes = useStyles();
+
+  const handleItem = (id, currentState = noteList, actionFunction) => {
+    const filtredNotes = actionFunction(id, currentState);
+    setNoteList(() => [...filtredNotes]);
```

Comment:
Why did you use arrow function inside `setNoteList`? Why did you add rest prop?
```suggestion
    setNoteList(filtredNotes);
```

Reply:
I thought that would use 'prev => ...prev' , but its isnt my case, after some time i refused it and.... forgot to fix